### DISCUSSION TITLE
Z Wave LR support

### DISCUSSION THREAD

**Discussion by notrackingme** [zwave-js/zwave-js#6190] [General] [answered]
Hi.  Any update on when LR (long range) will be supported?  Thank you.

**Comment by jtbraun** [comment]
@AlCalzone just released v12.5.0 beta that adds support for it in `node-zwave-js`. Some changes to applications are required to make use of it (IE: a "use long range inclusion" option when adding a node to the network)

**Reply by sabolcik** [reply]
Is anyone aware of a docker image for RPI that is built using 12.5.0?  It appears that the latest release on dockerhub is 12.4.4.  I tried pulling and building the 12.5.0 branch but I don't have any experience doing it and I was getting build errors tied to the version of nodejs on my machine.

**Reply by AlCalzone** [reply]
@robertsLando ?

**Reply by robertsLando** [reply]
I may have missed that release, coming now

**Reply by robertsLando** [reply]
Oh wait that's still on beta... In order to use it you have to use docker `test` tag or checking out this branch: https://github.com/zwave-js/zwave-js-ui/pull/3545

**Comment by AlCalzone** [comment, marked as answer]
12.5.x is out of beta
